cleanlab_studio/internal/enrichment_utils.py

```diff
@@ -9,11 +9,17 @@
 from cleanlab_studio.studio.studio import Studio
 from cleanlab_studio.studio.trustworthy_language_model import TLMResponse
 
+Replacement = Tuple[str, str]
```

Comment:
This is intentionally changed from Union[re.Pattern[str], str] in case user wants to pass in compiled patterns, right?

Reply:
Yes, we are no longer allowing compile regex patterns (only strings that we will compile at runtime)